simulation/traffic_simulator/include/traffic_simulator/hdmap_utils/hdmap_utils.hpp

```diff
@@ -35,9 +35,9 @@
 #include <geometry/spline/catmull_rom_spline_interface.hpp>
 #include <geometry/spline/hermite_curve.hpp>
 #include <geometry_msgs/msg/pose_stamped.hpp>
-#include <lanelet2_extension/utility/message_conversion.hpp>
-#include <lanelet2_extension/utility/query.hpp>
-#include <lanelet2_extension/utility/utilities.hpp>
+#include <autoware_lanelet2_extension/utility/message_conversion.hpp>
+#include <autoware_lanelet2_extension/utility/query.hpp>
+#include <autoware_lanelet2_extension/utility/utilities.hpp>
```

Comment:
These line will occur compilation errors on almost TIER IV internal projects that uses non-latest autoware.
You need to go back to `lanelet2_extension`.

Reply:
The same applies to other places where `autoware_lanelet2_extension` is used.

Reply:
I've reverted the modification.
https://github.com/tier4/scenario_simulator_v2/pull/1300/commits/17f873bf561d59c43db14a6c2bb6f552e03d51aa